### ISSUE TITLE
gRPC examples: Why are .proto in `impl` and not `api`?

### ISSUE THREAD

**Issue by ryanhanks** [lagom/lagom-samples#145]
Do the samples currently demonstrate best practices around where `.proto` files should live?

The examples currently don't share a schema definition between the client and the implementing service. Why is that? Is this not something that we would expect to see in the `api` package, and then used by both the service making the client call and the service providing the handler?

**Comment by octonato** [comment]
You are right about it. This should be added to the `api` module. I think this was just an oversight. 

Thanks for reporting this.

I added this issue to the backlog, but it can take sometime before someone can find time to pick it. 

Would you be interested in sending a PR?

**Comment by ignasi35** [comment]
Actually, I don't think that's supported at the moment. I thought we had a specific inssue in either `lagom/lagom` or playframework/play-grpc` (which includes lagom support for grpc) but I could only find  https://github.com/akka/akka-grpc/issues/152 (and related issues).
